scripts/lib/CIME/case.py

```diff
@@ -136,6 +136,7 @@ def initialize_derived_attributes(self):
             "compiler" : self.get_value("COMPILER"),
             "mpilib"   : self.get_value("MPILIB"),
             "threaded" : self.get_build_threaded(),
+            "queue": self.get_value("JOB_QUEUE", subgroup="case.run")
```

Comment:
This concerns me a lot as it appears to be a catch-22. Way down in the case creation code, it says that initialize_derived_attributes must be called before the env_batch file is properly set up. I think this is because queue choice depends on num_nodes. But it now looks like initialize_derived_attributes depends env_batch being set up already.

Reply:
We don't actually need this, I added it before tracking down the real problem.   Removed.